### DISCUSSION TITLE
dashboard中，节点置灰，改写订阅文件，添加跳过证书验证，但依然置灰，clash for windows上可正常使用

### DISCUSSION THREAD

**Discussion by Z46I4W5476SJA** [vernesong/OpenClash#3128] [Q&A]
我有一个节点，dashboard打开，此节点置灰，于是在配置文件加了跳过证书验证，但依然还是置灰，此节点在windows电脑的clash for windwos上可正常ping通  也可正常使用，当前版本哪怕更新到最新版本  依然如此，求各位大佬帮忙看看是什么问题，研究两三天了
![image](https://user-images.githubusercontent.com/45085951/226914126-2905dee2-b28d-463d-ba27-04ecf8c0e811.png)
![image](https://user-images.githubusercontent.com/45085951/226914206-d3d22f7d-ba3d-48f7-aa75-834c2253dc59.png)
![image](https://user-images.githubusercontent.com/45085951/226914361-ad2f0142-d0fb-401d-8954-ae40d017c5a3.png)

**Comment by Idealisten** [comment]
我也遇到了一样的问题，测试延迟节点是灰色的，但是这个节点明明可以用在其他app上都用的好好的。不知道openclash这个测试延迟是什么原理？

**Comment by Z46I4W5476SJA** [comment]
你是自建的还是机场的？自建的可以试试看添加跳过证书的内容   可能就可以了发自我的 iPhone在 2023年3月28日，01:18，Anonymous ***@***.***> 写道：﻿
我也遇到了一样的问题，测试延迟节点是灰色的，但是这个节点明明可以用在其他app上都用的好好的。不知道openclash这个测试延迟是什么原理？

**Comment by Idealisten** [comment]
我是自建的，跳过证书验证是不是会存在安全风险啊。而且我的情况是刚建好时候都没问题过几天才变灰色，可能哪天又好了，过几天又抽风了

chouxiang99 ***@***.***>于2023年3月28日 周二09:04写道：

**Comment by Z46I4W5476SJA** [comment]
跳过证书验证问题不大，或者你就等几天，我是弄了跳过证书后，依然灰色，几天没管，后来突然又好了发自我的 iPhone在 2023年3月28日，10:34，Anonymous ***@***.***> 写道：﻿
我是自建的，跳过证书验证是不是会存在安全风险啊。而且我的情况是刚建好时候都没问题过几天才变灰色，可能哪天又好了，过几天又抽风了



chouxiang99 ***@***.***>于2023年3月28日 周二09:04写道：

**Comment by Idealisten** [comment]
真的好迷，我还在配置文件里自己写了测试url但是效果反而更差了，在网上也没搜到相关信息，很想知道这个测试延迟的原理是什么以及在哪里设置
chouxiang99 ***@***.***>于2023年3月28日 周二10:42写道：

**Comment by Z46I4W5476SJA** [comment]
这个原理我也不知道了   issue里面找找呢？实在不行就放几天吧…玄学  真的玄学发自我的 iPhone在 2023年3月28日，11:01，Anonymous ***@***.***> 写道：﻿
真的好迷，我还在配置文件里自己写了测试url但是效果反而更差了，在网上也没搜到相关信息，很想知道这个测试延迟的原理是什么以及在哪里设置

chouxiang99 ***@***.***>于2023年3月28日 周二10:42写道：
